### PR TITLE
Cleanup asset handling

### PR DESCRIPTION
**Pull request by justinsb**
Start using assets everywhere, in preparation for hash-based discovery.

### CODE REVIEW

File: upup/pkg/fi/cloudup/urls.go [hunk 1]
```diff
@@ -89,96 +82,96 @@ func SetKopsAssetsLocations(assetsBuilder *assets.AssetBuilder) error {
...
 	}
 
-	return protokubeLocation, protokubeHash, nil
+	return protokubeAsset, nil
 }
 
-// KopsFileUrl returns the base url for the distribution of kops - in particular for nodeup & docker images
-func KopsFileUrl(file string, assetBuilder *assets.AssetBuilder) (*url.URL, *hashing.Hash, error) {
-	base, err := BaseUrl()
+// KopsAsset returns an for a file from the distribution of kops - in particular for nodeup & docker images
```
Comment (rdrgmnzs) [inline upup/pkg/fi/cloudup/urls.go]: nit: Comment needs to be fixed.

**Comment by rdrgmnzs** [comment]
@justinsb mind rebasing to pull in #5893 for testing?

**Comment by justinsb** [comment]
Rebased and fixed up comment, but also moved to milestone 1.12

**Comment by justinsb** [comment]
Rebased but moving to 1.13 - just cleanup

**Pull request closed without merging**